common/src/main/java/org/opensearch/ml/common/transport/batch/MLBatchIngestionInput.java

```diff
@@ -28,13 +28,17 @@ public class MLBatchIngestionInput implements ToXContentObject, Writeable {
 
     public static final String INDEX_NAME_FIELD = "index_name";
     public static final String FIELD_MAP_FIELD = "field_map";
-    public static final String DATA_SOURCE_FIELD = "data_source";
+    public static final String INGEST_FIELDS = "ingest_fields";
```

Comment:
What does `INGEST_FIELDS` mean? May be add a comment as this field is not self explanatory?